### PR TITLE
Protax improvements 176179408

### PR DESCRIPTION
**Pull request by aorin**
Improvements to protax page. Testable [here](https://176179408.dev.laji.fi/theme/protax).

### CODE REVIEW

File: projects/laji/src/app/+theme/protax/protax-form/protax-form.component.ts [hunk 1]
```diff
@@ -35,6 +38,11 @@ export class ProtaxFormComponent {
   }
 
   submitForm() {
+    if (this.probabilityThreshold == null || this.probabilityThreshold < 0 || this.probabilityThreshold > 1) {
```
Comment (vixriihi) [inline projects/laji/src/app/+theme/protax/protax-form/protax-form.component.ts:41]: `this.probabilityThreshold == null` could be replaced with just `!this.probabilityThreshold` or if you need to check the null case specifically you need to use `this.probabilityThreshold === null`

**Review by vixriihi — CHANGES_REQUESTED** [review]
small tweak suggestions 👍

File: projects/laji/src/app/+theme/protax/protax-form/protax-form.component.ts (hunk 1 above)
Comment (aorin) [reply projects/laji/src/app/+theme/protax/protax-form/protax-form.component.ts:41]: `!this.probabilityThreshold` would not work with 0 so I used `this.probabilityThreshold == null` to check that it isn't undefined or null

**Comment by aorin** [comment]
@vixriihi now the alerts use dialog service and ngif repetition is fixed

File: projects/laji/src/app/+theme/protax/protax-form/protax-form.component.ts (hunk 1 above)
Comment (vixriihi) [reply projects/laji/src/app/+theme/protax/protax-form/protax-form.component.ts:41]: true 👍

**Review by vixriihi — APPROVED** [review]
LGTM. Should this be included with the next release or is it enough to go in the one after?

**Pull request merged**